stable/cert-manager/Chart.yaml

```
@@ -1,5 +1,5 @@
 name: cert-manager
-version: v0.5.2
+version: v0.5.3
```

```suggestion
version: v0.6.0
```
since it adds a feature